### PR TITLE
Windows: Remove windowsdiff graph driver

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com

Removes support for the windowsdiff graph driver. Platform support hasn't been inbox for many months now. 

@swernli

### CODE REVIEW

File: daemon/graphdriver/windows/windows.go [hunk 1]
```diff
@@ -79,22 +69,15 @@ func InitDiff(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (
 	d := &Driver{
```
Comment (swernli) [inline daemon/graphdriver/windows/windows.go]: Why not remove InitDiff entirely?

Comment (jhowardmsft) [inline daemon/graphdriver/windows/windows.go]: No idea how I missed that. Removed.

**Comment by cpuguy83** [comment]
LGTM

File: daemon/graphdriver/windows/windows.go [hunk 2]
```diff
@@ -67,34 +56,15 @@ func InitFilter(home string, options []string, uidMaps, gidMaps []idtools.IDMap)
...
-}
-
-// InitDiff returns a new Windows differencing disk driver.
-func InitDiff(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (graphdriver.Driver, error) {
-	logrus.Debugf("WindowsGraphDriver InitDiff at %s", home)
-	d := &Driver{
-		info: hcsshim.DriverInfo{
-			HomeDir: home,
-			Flavour: diffDriver,
+			Flavour: 1,
```
Comment (runcom) [inline daemon/graphdriver/windows/windows.go]: I'd rather keep a const for this

**Comment by runcom** [comment]
LGTM except a nit

**Comment by jhowardmsft** [comment]
@runcom Ooops, missed your comment ^^ previously. Nit addressed.

**Comment by vdemeester** [comment]
LGTM 🐯

**Comment by thaJeztah** [comment]
@jhowardmsft is this something for the changelog and/or deprecated features list? I'm not sure we've communicated these options, so wondering if we should mention

**Comment by jhowardmsft** [comment]
No need - was only ever used internally during development.

**Comment by thaJeztah** [comment]
Cool, thanks!

**Pull request merged**